components/layouts/MultiLayout.js

```diff
@@ -1,12 +1,18 @@
 import Navbar from "@components/navbar/Navbar";
 import Footer from "@components/Footer";
+import { SkipLink } from "@components/SkipLink";
```

Comment:
When updating the component style, this will need to change also

```suggestion
import SkipLink from "@components/SkipLink";
```